tox.ini

```diff
@@ -35,3 +35,15 @@ deps =
     pylint
 whitelist_externals =
     sh
+
+[testenv:linters]
+skip_install = true
+basepython = python3
+deps = flake8
+       flake8-colors
+       black
+commands =
+    # do not fail for black
+    -black --skip-string-normalization --check .
```

Comment:
Looks like it passes black, so you can remove the `-`

Reply:
I get this when running black
```
All done! 💥 💔 💥
176 files would be reformatted, 101 files would be left unchanged.
ERROR: InvocationError for command '/home/pcm/dev/wazo/wazo-dird/.tox/linters/bin/black --skip-string-normalization --check .' (exited with code 1)
___________________________________________________________________________________________________________________ summary ___________________________________________________________________________________________________________________
ERROR:   linters: commands failed

```

Reply:
Oh ok, so just run 'tox -eblack', commit all changed files, and the you can remove it :)

Reply:
done